<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>playerResultWidget</class>
 <widget class="QWidget" name="playerResultWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>playerResultWidget</string>
  </property>
  <widget class="QLabel" name="playerNameLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>-20</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Player Name</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="archetypeLabel">
   <property name="geometry">
    <rect>
     <x>68</x>
     <y>50</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Archetype</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="rarityLabel">
   <property name="geometry">
    <rect>
     <x>71</x>
     <y>70</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>System</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Rarity</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="backstoryLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>281</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string>This is where the player's backstory will go. It will describe their origins, along with many interesting stories which led them to become the person they are today. These are generated by ChatGPT 4.0, and deliver esquisite tales which are faction-based and varied by a multitude of modifiers. So, sit right down and enjoy a heart-warming backstory about Spritopias newest basketball talent. This next sentence is purely to fill more space, as I am trying to see how much room I need to provide to create adequate space for these backstories, but also not too much to leave tons of blank, unusable space where other fun and interesting things can be put.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="backstorySeparater">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="factionIcon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>34</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>FactionIcons/Sludgeservants.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="artifactIcon">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>34</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>FactionIcons/American Illegals.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="infoTabs">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>281</width>
     <height>471</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="attributesTab">
    <attribute name="title">
     <string>Attributes</string>
    </attribute>
    <widget class="QFrame" name="primaryAttributesHolder">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>271</width>
       <height>108</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="secondaryAttributesHolder">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>271</width>
       <height>108</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="tertiaryAttributesHolder">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>220</y>
       <width>271</width>
       <height>108</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="otherAttributesHolder">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>330</y>
       <width>271</width>
       <height>108</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="statsTab">
    <attribute name="title">
     <string>Stats</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="FactionIcons/FactionIcon.qrc"/>
 </resources>
 <connections/>
</ui>
